kctf-operator/pkg/apis/kctf/v1alpha1/challenge_types.go

```diff
@@ -102,27 +109,32 @@ type ChallengeSpec struct {
 	PodTemplate *corev1.PodTemplate `json:"podTemplate"`
 
 	// PersistentVolumeClaim are used to determine how much resources the author requires for its challenge
```

Comment:
Also we should change the comment because we don't specify the resources here; just provide the names for the volumes